### ISSUE TITLE
Broken Typings

### ISSUE THREAD

**Issue by danielsogl** [gpbl/react-day-picker#915]
The release 7.3.1 has wrong TypeScript definitions.

Build errors:

```

node_modules/react-day-picker/types/index.d.ts:10:35 - error TS2694: Namespace '"/Users/danielsogl/workspace/vda-rga/react-base-components/node_modules/react-day-
picker/types/utils"' has no exported member 'DateUtils'.

10 export const DateUtils: UtilTypes.DateUtils;
                                     ~~~~~~~~~

node_modules/react-day-picker/types/index.d.ts:11:37 - error TS2694: Namespace '"/Users/danielsogl/workspace/vda-rga/react-base-components/node_modules/react-day-
picker/types/utils"' has no exported member 'LocaleUtils'.

11 export const LocaleUtils: UtilTypes.LocaleUtils;
                                       ~~~~~~~~~~~

node_modules/react-day-picker/types/index.d.ts:12:40 - error TS2694: Namespace '"/Users/danielsogl/workspace/vda-rga/react-base-components/node_modules/react-day-picker/types/utils"' has no exported member 'ModifiersUtils'.

12 export const ModifiersUtils: UtilTypes.ModifiersUtils;
```

**Comment by deju** [comment]
@danielsogl 
after remove line 10,11,12, the lib works without errors.

**Comment by omcg33** [comment]
With errors begins after 3.7.1 version. in types/index.d.ts where are wrong exports of utils. 
Typescript 3.5.1

**Comment by adam-lebon** [comment]
Since the last version `DateUtils`, `LocaleUtils` and `ModifiersUtils` are not interfaces anymore.
They should be exported like this : 
```typescript
export const DateUtils: typeof UtilTypes.DateUtils;
export const LocaleUtils: typeof UtilTypes.LocaleUtils;
export const ModifiersUtils: typeof UtilTypes.ModifiersUtils;
```

**Comment by gpbl** [comment]
Hey folks, I cannot follow the typescript part anymore and I'm thinking to remove types from this library. Developers here continue to submit wrong PRs about typings and since I'm not a TypeScript user I have no time to maintain this.

Please decide what is the right typings you want and make them work :) 

I'm trying to fix this here:
https://github.com/gpbl/react-day-picker/pull/916

but I'm running into issues with `dtslint`.

**Comment by adidahiya** [comment]
this should be fixed in v7.2.0 by #929
